packages/synapse-core/tools/endorse-sp.js

```diff
@@ -1,19 +1,28 @@
+import EthModule from '@ledgerhq/hw-app-eth'
+import TransportNodeHidModule from '@ledgerhq/hw-transport-node-hid'
 import { createWalletClient, http } from 'viem'
-import { privateKeyToAccount } from 'viem/accounts'
+import { privateKeyToAccount, toAccount } from 'viem/accounts'
+
+const TransportNodeHid = TransportNodeHidModule.default || TransportNodeHidModule
```

Comment:
I don't think this is needed, this looks like it's either making allowance for dodgy bundlers (not a problem here) or dealing with an older version of the module. According to https://github.com/LedgerHQ/ledger-live/tree/develop/libs/ledgerjs/packages/hw-transport-node-hid, we should just be able to use the main export `import TransportNodeHid from "@ledgerhq/hw-transport-node-hid";` - i.e. get rid of `Module` on the end of the import above and remove this line.